src/styles/common/adaptive.scss

```diff
@@ -0,0 +1,3 @@
+$mobile-max-width: 768px;
```

Comment:
I guess it's better to use `767px` as `$mobile-max-width` and `768px` as a minimum tablet width